src/app/movies/[id]/starRating.tsx

```diff
@@ -7,18 +8,41 @@ interface Props {
 }
 
 export const StarRating = ({ rating, setRating, size = 30 }: Props) => {
+  const [hoverRating, setHoverRating] = useState<number | null>(null);
```

Comment:
Mietitkö voisko tätä tähtien hover efektiä saada puhtaalla css aikaiseksi?

Reply:
Esim. https://stackoverflow.com/questions/21196588/css-hover-effect-on-current-and-previous-elements

Reply:
En miettinyt, tuota vois kyllä kokeilla!